resources.html

```diff
@@ -99,18 +99,72 @@ <h5>At a glance</h5>
             </ul>
         </div>
+
+        <h3>Tips</h3>
+
+        <p>
+            Compression changes how we need to think about size coding. Reducing the size of the compiler output doesn't
+            mean the compressed file will be smaller. If you're short on space, we recommend that you check the compressed size
+            regularly to avoid surprises (e.g. you can include it in the commit messages).
+        </p>
+        <p>Some techniques can be useful:</p>
+        <ul>
+            <li>Group data by type, so that similar things are packed together.
+            <li>Repetitive data doesn't take much space. Sometimes, the smart "Don't Repeat Youself" principle can be
+                counterproductive.
+            <li>Try <a href="https://en.wikipedia.org/wiki/Delta_encoding">delta encoding</a>.
+            <li><a href="https://www.ctrl-alt-test.fr/2018/making-floating-point-numbers-smaller/">Make the floats smaller</a>.
+        </ul>
+        You may also read <a href="https://conspiracy.hu/articles/6/">Why are SMF files smaller?</a> about packing the music
+        data.
     </div>
 
     <div class="card">
         <h2>Music</h2>
         <p>It's common for the music to take around 16kB, but this can vary a lot. Instruments are often generated using sound
             synthesis, and the list of notes is compressed.
         </p>
+        <p>
+            Multiple synths have been published that were specifically designed for 64k intros:
+        </p>
+        <ul>
+            <li><a href="https://github.com/hzdgopher/64klang">64klang</a>, an extended version of 4klang (a popular synth for 4k
+                intros).
+            <li><a href="https://github.com/logicomacorp/WaveSabre">WaveSabre</a>, used in Logicoma intros and many others.
+            <li><a href="https://tunefish-synth.com/">Tunefish</a>, used in Brain Control intros.
+            <li><a href="http://1337haxorz.de/products.html">v2</a> was a popular option around 2005-2015. It uses less
+                CPU than the alternatives above, but it can feel slightly outdated.
+        </ul>
+    </div>
+
+    <div class="card">
+        <h2>Texture</h2>
+
+        <p>
+            Procedural texture generation can be done either on the CPU or on the GPU. The article
+            <a href="https://conspiracy.hu/articles/9/">GPU Based Texture Synthesis</a> discusses some advantages of using the GPU.
+            Generating textures on the CPU can be slow, so you'll probably want to use multiple threads.
+        </p>
+
+        <p>General introduction to texture generation:</p>
+        <ul>
+            <li><a href="http://www.upvector.com/?section=Tutorials&subsection=Intro%20to%20Procedural%20Textures">Intro to
+                    Procedural Textures</a>
+            <li><a href="https://www.ctrl-alt-test.fr/2018/texturing-in-a-64kb-intro/">Texturing in a 64kB intro</a>, an overview of
+                the texture generation in Immersion.
```

Comment:
```suggestion
            <li><a href="https://www.ctrl-alt-test.fr/2018/texturing-in-a-64kb-intro/">Texturing in a 64kB intro</a>, an overview of
                the texture generation in H – Immersion.
```
But removing the mention altogether works as well.